cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BrokerStats.java

```diff
@@ -181,4 +186,17 @@ public String toString() {
     }
     return sb.toString();
   }
+
+  /**
+   * Format the double type CORE_NUM to int if it is not a fractional value, otherwise format it to double with 2 digits after decimal point
+   * @param numCores the number of cores
+   * @return StringFormatter String for the CORE_NUM
+   */
+  private String coreNumStringFormatter(double numCores) {
+    if (numCores == (int) numCores) {
```

Comment:
This works for values in the int range, which is the expected case for numCores -- but to make it even more generic regardless of the input size, would it make sense to use the following?
```
(numCores % 1) == 0
```

something like:
```
  private static String coreNumStringFormatter(double numCores) {
    return (numCores % 1) == 0 ? "%19.0f" : "%19.2f";
  }
```

Reply:
Yeah I implement it that way exactly because coreNum should fit in int value, but I'm totally ok to change it to be more generic